Model/Resolver/Reorder.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Scandipwa\SalesGraphQl\Model\Resolver;
+
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlAuthorizationException;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\GraphQl\Model\Query\ContextInterface;
+use Magento\Sales\Model\Reorder\Data\Error;
+use Magento\Sales\Model\OrderFactory;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\SalesGraphQl\Model\Resolver\Reorder as CoreReorder;
+use Magento\Sales\Model\Reorder\Reorder as ResolveReorder;
+
+/**
+ * ReOrder customer order
+ */
+class Reorder extends CoreReorder
+{
+    /**
+     * Order number
+     */
+    public const ARGUMENT_ORDER_NUMBER = 'orderNumber';
+
+    /**
+     * @var OrderFactory
+     */
+    public $orderFactory;
+
+    /**
+     * @var ResolveReorder
+     */
+    public $reorder;
+
+    /**
+     * @param ResolveReorder $reorder
+     * @param OrderFactory $orderFactory
+     */
+    public function __construct(
+        ResolveReorder $reorder,
+        OrderFactory $orderFactory
+    ) {
+        parent::__construct(
+            $reorder,
+            $orderFactory
+        );
+
+        $this->orderFactory = $orderFactory;
+        $this->reorder = $reorder;
+    }
+
+    /**
+     * @inheritDoc
```

Comment:
could use a comment on the purpose of the rewrite
which was to remove the store id filter from the order load